server/apps/content_filters/content_filter_tests.py

```diff
@@ -312,7 +312,7 @@ def test_build_elastic_query_using_like_filter_multi_content_filter4(self):
 
     def test_does_match_returns_false_for_nonexisting_filter(self):
         for article in self.articles:
-            self.assertFalse(self.f.does_match(None, article))
+            self.assertTrue(self.f.does_match(None, article))
```

Comment:
The name of this test case method should be updated as well to avoid confusion. :)